azure/Guidelines.md

```diff
@@ -757,68 +761,92 @@ implemented as a _long-running operation (LRO)_. This allows clients to continue
+
+:white_check_mark: **DO** include an `Operation-Id` header in the response with the ID of the status monitor for the operation.
 
-:white_check_mark: **DO** include a `Retry-After` header in the response to the initiating request and requests to the operation-location URL. The value of this header should be an integer number of seconds to wait before making the next request to the operation-location URL.
+:white_check_mark: **DO** include response headers with any additional values needed for a GET request to the status monitor (e.g. location).
 
-:heavy_check_mark: **YOU MAY** support a `get` method on the status monitor collection URL that returns a list of status monitors for all recently initiated operations.
+:white_check_mark: **DO** include a `Retry-After` header in the response to the initiating request if the operation is not complete. The value of this header should be an integer number of seconds to wait before making the first request to the status monitor.
 
-:warning: **YOU SHOULD NOT** return any other `2xx` status code from the initial request of a status-monitor LRO -- return `202-Accepted` and a status monitor URL even if processing was completed before the initiating request returns.
+:ballot_box_with_check: **YOU SHOULD** include an `Operation-Location` header in the response with the absolute URL of the status monitor for the operation, but do not include an api-version query parameter.
 
-:no_entry: **DO NOT** return any data in the response body of a `202-Accepted` response.
+#### Obtaining status and results of long-running operations
 
-Previous Azure guidelines specified "Azure-AsyncOperation" as the name of the response header containing the status monitor URL.
+For all long-running operations, the client will issue a GET on a status monitor resource to obtain the current status of the operation.
 
-:white_check_mark: **DO** return **both** `Azure-AsyncOperation` and `Operation-Location` headers if your service previously returned `Azure-AsyncOperation`, even though they are redundant, so that existing clients will continue to operate.
+:white_check_mark: **DO** support the GET method on the status monitor endpoint that returns a `200-OK` response with the current state of the status monitor.
 
-:white_check_mark: **DO** return the same value for **both** headers.
+:white_check_mark: **DO** return a status monitor in the response body that conforms with the following structure:
 
-:white_check_mark: **DO** look for **both** headers in client code, preferring the `Operation-Location` header.
+**OperationStatus** : Object
+
+Property | Type        | Required | Description
+-------- | ----------- | :------: | -----------
+`id`     | string      | true     | The unique id of the operation
+other    |             | true     | Other values needed for a GET request to the status monitor (e.g. location)
```

Comment:
Wondering if we should be more clear about this, e.g. if it should be an object, e.g. `additionalProperties`. Does this help with breaking changes, e.g. if I need to add something in the future?